### DISCUSSION TITLE
UN Datathon 2023

### DISCUSSION THREAD

**Discussion by ShweataNHegde** [petermr/petermr#74] [Planning]
* 5-person hybrid team synchronous and asynchronous volunteers spread over India, UK, and Germany.
*(Mainly async)
* Leads are young Indians:
  - Shweata Hegde, team lead and coder (IN and in-person in UniGE)
  - Renu Kumari, Project Manager (IN at NIGPR), online
  - Gayatri Taneja, (IN at school) online
  - Ayush Garg (India, online in US)
* Support 
  - Simon Worthington (DE, in person). TiB German National Informatics Library and NFDI (German informatics program)
  - Peter Murray-Rust, co-founder, Semantic Climate, coder, and Wikimedia liaison
 * Team brings:
   - tools for semantic climate (especially UN IPCC AR6, 10,000 pp)
   - novel tools to access the open scholarly literature and extract knowledge

Without knowing the precise challenge (yet!) we would expect to add value by:
  - adding semantics to challenge datasets, and especially climate annotation from IPCC
  - real-time queries of the current open research in the challenge themes

We believe these will be novel and will be complementary to what others bring so we are happy to collaborate at any stage.

A strong theme is that young volunteers (school, and early Uni) have built much of the toolkit and also presented to several groups this year (FSCI2023, CODATA in Beijing, Berlin Barcamp and Wikimedia).

**Comment by ShweataNHegde** [comment]
### Questions to answer
1. Why do you wish to participate in the 2023 UN Datathon?
We've been supported by SDG Accelerator, University of Geneva, to submit a proposal for the Datathon. They are supporting Shweata, the team lead, to attend the event in Geneva, one of the local satellites. Our team of young scientists from India, #semanticClimate, has been working intensively on extracting climate data from semi-structured resources such as IPCC Reports and the open-access climate literature. As the UN Secretary-General said, "We have never been better equipped to solve the climate challenge – but we must move into warp speed climate action now". To move at warp speed, we need data. So, we are developing our open tools in Python to create semantic climate glossaries (in triple form) and enhance them with sources such as Wikidata (an open database). These can be used to annotate a wide range of documents with climate knowledge (SDG 13). 

We have also extracted geospatial information from the scientific literature. We believe this will complement the geospatial datasets the Datathon provides and add an extra dimension to their interpretation and use. 

Our tools and end-products are designed specifically to be accessed by anyone (high school and beyond). 

2. Do you have experience working with geospatial data? If so, please provide a brief description of your work.
Yes. Much openly available science literature contains geospatial text-- like cities, regions, rivers, etc. However, they aren't accessible since they are unstructured. We convert text names to coordinates. We use this structured coordinate data to represent scientific information on maps. 

3. Do you have experience in data science? If so, please provide a brief description of your work.
Yes. Peter has worked for 50 years on chemical data and, more generally, informatics (developer of XML). Simon works in the TiB running open science. Ayush developed a semantic data extraction tool for corpora when he was in high school and is now getting his computer science degree in the US. To complement this, Shweata has developed a tool to extract data (such as GPE) from individual documents. 

_Please note that previous experience in data science is not required to participate in the Datathon as a beginning team_

**Reply by gayatritaneja** [reply]
> **These can used** to annotate a wide range of documents with climate knowledge (SDG 13). 

Just a minor typo, "be" seems to be missing.

**Reply by ShweataNHegde** [reply]
Thanks!

**Comment by petermr** [comment]
Thanks Gayatri! Well spotted.
These are also directly editable on the Github website.

On Tue, Sep 26, 2023 at 2:19 PM Gayatri Taneja ***@***.***>
wrote:

> *These can used* to annotate a wide range of documents with climate
> knowledge (SDG 13). We have also extracted geospatial information from
> these resources.
>
> Just a minor typo, "be" seems to be missing.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/petermr/petermr/discussions/74#discussioncomment-7112700>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFTCS33ABOITXPYMW3YDOLX4LI5XANCNFSM6AAAAAA5HOS76Y>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Peter Murray-Rust
Founder ContentMine.org
and
Reader Emeritus in Molecular Informatics
Dept. Of Chemistry, University of Cambridge, CB2 1EW, UK

**Comment by mrchristian** [comment]
Have we received the confirmation of them formally accepting our submission.
